src/common/snippets/src/lowered/loop_manager.cpp

```diff
@@ -272,7 +272,7 @@ void LoopManager::fuse_loops(LinearIR::constExprIt loop_begin_target, LinearIR::
     auto input_ports_upper = loop_info_upper->get_input_ports();
     auto output_ports_upper = loop_info_upper->get_output_ports();
     auto input_ports_lower = loop_info_lower->get_input_ports();
-    auto output_ports_lower = loop_info_lower->get_output_ports();
+    const auto& output_ports_lower = loop_info_lower->get_output_ports();
```

Comment:
Ok, I might guess that `input_ports_upper` and `output_ports_upper` are used by value because we want to `std::move` them later. But what is the difference between `input_ports_lower` and `output_ports_lower`? Why the former is used by value, and the latter - by const reference?